toolkit/javascripts/analytics/_trackExternalLinks.js

```diff
@@ -0,0 +1,19 @@
+(function (GOVUK) {
+  GOVUK.GDM.analytics.trackExternalLinks = {
+    'init': function () {
+      this.trackExternalLinks();
+    },
+
+    trackExternalLinks: function () {
+      $('a').filter(function () {
+        return this.hostname && this.hostname !== location.hostname;
```

Comment:
Do we want to count e.g. S3 asset links as 'external'? Document download links on a supplier service page would be an example.

Reply:
Looking at the current analytics code in the Buyer FE, we have a function to determine whether a link `isOnHostDomain`:

```
 LinkClick.prototype.isOnHostDomain = function (url) {
      var currentHost = GOVUK.GDM.analytics.location.hostname(),
          currentHostRegExp = (currentHost !== '') ? new RegExp(currentHost) : /^$/g;

      return url.match(currentHostRegExp) !== null;
 };
```
I'm not sure what's in `GOVUK.GDM.analytics.location.hostname` - should we be using this instead of `location.hostname`?

Reply:
that function basically just returns the same, I'll update it for consistency 
```
hostname: function() {
                return e.location.hostname
 }```